examples/linear_model/plot_sparse_logistic_regression_20newsgroups.py

```diff
@@ -18,6 +18,7 @@
 input features would be to use univariate feature selection followed by a
 traditional (l2-penalised) logistic regression model.
 """
+from __future__ import division
```

Comment:
I like the `__future__` but there are advantages in the other, like not confusing python 2 users who are unaware of the `__future__`, making copy-paste work. If Alex suggested it, I wouldn't seek a second opinion, either ;)